### ISSUE TITLE
Unexpected closing tag "__MockOfEmptyOutletComponent-selector" still appearing in 7.1

### ISSUE THREAD

**Issue by stephenwil** [getsaf/shallow-render#78]
Loved the idea of this project, but can't use in the most simplest of tests of a child component, which is contained in a module that has routing.
None of the recommendations work for the issue previously, and the stackblitz examples fail with the same error.

https://stackblitz.com/edit/github-frqbzt-dek4h5?file=examples%2Fcomponent-with-routing.spec.ts

**Comment by getsaf** [comment]
That stackblitz was using shallow-render 6.1.

I upgraded to `shallow-render&7.1` and fixed the `beforeEach` to be:
```typescript
  beforeEach(() => {
    shallow = new Shallow(GoHomeLinkComponent, GoHomeModule)
      .replaceModule(GoHomeRoutingModule, RouterTestingModule.withRoutes(routes));
  });
```

Those test should run with this pattern!

**Comment by getsaf** [comment]
A quick note: The *official* Stackblitz is updated with all the [example specs from shallow-render](https://github.com/getsaf/shallow-render/tree/master/lib/examples) every time I add a new feature or enhance something that should be considered "best practice" when writing shallow specs.

I'm not sure how you got such an old Stackblitz project (6.1 was released in early August and was outdated by the end of August).

Using the routing module has been one of the biggest PITAs, but with the help of some folks on the issues section, we got a pretty simple solution in place. Hopefully, version 7.1 has the fix you need to make this useful for your project.

**Comment by getsaf** [comment]
Addressed in #82 

If you have a chance to give this new version a go, let me know if it works and close out the issue!
